### PR TITLE
 Added Read and Write with edgeType under Edges

### PR DESCRIPTION
**Pull request by nimeshawij**
Added 

- Read edges with (aka connection definition) in your data model 
- Write edges with (aka connection definition) in your data model

https://github.com/cognitedata/cdp-spark-datasource/edit/master/README.md#edges

Tested and documented here https://docs.google.com/document/d/1RJqEA0zHG2mfGPE8tL7vRyuxU7So2XrSt9qQ9iaEq7s/edit#heading=h.vs89qerrrdkq

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1306,32 +1306,66 @@ df = spark.read
...
+  .option("instanceType","edge")
+  .option("type", "instances")
+  .option("edgeTypeSpace", edgeTypeSpace)
+  .option("edgeTypeExternalId", edgeTypeExternalId)
+  .option("collectMetrics", true)
+  .load()
+
+df.show()
 
+# Write edges with (aka connection definition) in your data model
```
Comment (SunilVenekarKrishnamoorthy) [inline README.md:1353]: Write edges without view (aka connection definition) in your data model

File: README.md [hunk 2]
```diff
@@ -1306,32 +1306,66 @@ df = spark.read
...
     .option("instanceType","node") \
     .option("type", "instances") \
     .option("space", ViewSpace) \
-    .option("externalId", ViewExternaldId) \
+    .option("externalId", ViewExternalId) \
     .option("existingVersion", existingVersion) \
     .option("onConflict", "upsert") \
     .save() \
 
+# Read edges with (aka connection definition) in your data model
```
Comment (SunilVenekarKrishnamoorthy) [inline README.md:1334]: Read edges without view (aka connection definition) in your data model

**Review by SunilVenekarKrishnamoorthy — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (nimeshawij) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (nimeshawij) [reply README.md]: done

**Pull request merged**